packages/cells/src/model.ts

```diff
@@ -412,13 +429,52 @@ export abstract class AttachmentsCellModel extends CellModel {
     return this._attachments;
   }
 
+  /**
+   * Dispose of the resources held by the model.
+   */
+  dispose(): void {
+    if (this.isDisposed) {
+      return;
+    }
+    this._attachments.stateChanged.disconnect(this.onGenericChange, this);
+    this._attachments.changed.disconnect(this._onAttachmentsChange, this);
+    this.sharedModel.changed.disconnect(this._onSharedModelChanged, this);
```

Comment:
The attachments container does not seem to be disposed:

```suggestion
    this._attachements.dispose();
    this.sharedModel.changed.disconnect(this._onSharedModelChanged, this);
```